src/lighteval/metrics/llm_as_judge.py

```diff
@@ -119,7 +119,7 @@ def __lazy_load_client(self):
                     from transformers import AutoModelForCausalLM, AutoTokenizer, pipeline
 
                     transformers_model = AutoModelForCausalLM.from_pretrained(
-                        self.model, torch_dtype=torch.bfloat16, trust_remote_code=False, device_map="cuda"
+                        self.model, torch_dtype=torch.float16, trust_remote_code=False, device_map="cuda"
```

Comment:
interesting, why is it needed?

Reply:
Because in google collab we cannot use blfoat16 :/

Reply:
:O